themes/ddbasic/scripts/menu.js

```diff
@@ -163,15 +163,15 @@
     // If there are too many links in the main menu, so it breaks into
     // two lines, we need to tell CSS to adjust.
     checkMainLinksOffset: function(context) {
-      var main_menu_links = $('.main-menu > li', context);
+      var main_menu_links = $('.main-menu-wrapper > .main-menu > li', context);
 
       // We have less than 2 elements, it doesnt make sense to compare anything.
       if (main_menu_links.length < 2) {
         return;
       }
 
       // Checking if the first and last elements are on the same line.
-      if (main_menu_links.first().offset().top != main_menu_links.last().offset().top) {
+      if (main_menu_links.first().offset().top !== main_menu_links.last().offset().top) {
```

Comment:
@guddo JS strict comparison is requested by coding standards.

Reply:
Let's leave this for ding2 team to fix.